test/spec/components/TextArea.spec.js

```diff
@@ -435,33 +437,6 @@ describe('<TextArea>', function() {
 
   describe('auto resize', function() {
 
-    it('should resize initially', function() {
```

Comment:
Removed this test since there is a valid use case when text area doesn't need to resize on initial render. When it's initially rendered inside of a closed group/collapsable, it will resize when it becomes visible.